javascript/chronometer.js

```diff
@@ -1,26 +1,57 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
+    console.log('startClick called');
```

Comment:
Just a small thing, but you may want to start removing console.logs when you submit labs or projects :)